share/functions/__fish_shared_key_bindings.fish

```diff
@@ -98,7 +98,7 @@ function __fish_shared_key_bindings -d "Bindings shared between emacs and vi mod
     bind --preset $argv \ed 'set -l cmd (commandline); if test -z "$cmd"; echo; dirh; commandline -f repaint; else; commandline -f kill-word; end'
     bind --preset $argv \cd delete-or-exit
 
-    bind --preset $argv \es "fish_commandline_prepend sudo"
+    bind --preset $argv \es "if command -v sudo; fish_commandline_prepend sudo; else if command -v doas; fish_commandline_prepend doas; end"
```

Comment:
`command -v` *prints* the command, so this would spew a message on screen. This should be `command -q sudo` (and `command -q doas`).

Reply:
Ah, I see Mahmoud already said as much. Oh well.

Reply:
sorry, instictively typed `command -v` since I use it *way too much*